<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Need For Speed Car Database</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="lwMerk">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>191</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lwModel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>191</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbEenheid">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>390</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Kies een eenheid</string>
    </property>
    <widget class="QRadioButton" name="btPK">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;PK</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="btKW">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;kW</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="lwVermogen">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>410</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lwLocatie">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lMerkPlaatje">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>331</width>
      <height>241</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lAutoMerk">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>AutoMerk</string>
    </property>
   </widget>
   <widget class="QLabel" name="lModel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="lLocatie">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Locatie</string>
    </property>
   </widget>
   <widget class="QLabel" name="lVermogen">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>390</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Vermogen</string>
    </property>
   </widget>
   <widget class="QLabel" name="lHomePlaatje">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>651</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>NFS images/Need_For_Speed_logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
